### ISSUE TITLE
Terrible name?

### ISSUE THREAD

**Issue by matthew-brett** [nipy/register#1]
I think 'register' is not the right name for this, because it's going to be a Python package, and it's not for general registration, it's for brain image registration - right?

**Comment by alexis-roche** [comment]
Hi Matthew,

That was just a first try to see if I could easily strip this out of nipy.
I was planning to let you know about that new package once a bit
stabilized. I agree that 'register' is a crappy name, although some of the
algorithms therein are general enough to work with non-brain images. Any
suggestion welcome.

Cheers,

Alexis

On Tue, May 26, 2015 at 6:23 PM, Matthew Brett <notifications@github.com>
wrote:

> I think 'register' is not the right name for this, because it's going to
> be a Python package, and it's not for general registration, it's for brain
> image registration - right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/register/issues/1>.
>



-- 
Lead Clinical Research
Advanced Clinical Imaging Technology
Siemens/CHUV/EPFL
1015 Lausanne, Switzerland
Phone: +41 21 545 9972
https://sites.google.com/site/alexisroche

**Comment by alexis-roche** [comment]
@matthew-brett Any new thoughts here? Do you like `nireg`?

**Comment by matthew-brett** [comment]
Yes nireg is fine.   I personally like 'reggie' because it makes me smile, but I know not everyone shares my taste.

**Comment by alexis-roche** [comment]
I am not sure I share your taste for `reggie` indeed. If you accept `nireg`, then `nireg` it will be.
